libPhoneNumberShortNumber.podspec

```diff
@@ -0,0 +1,22 @@
+Pod::Spec.new do |s|
+  s.name         = "libPhoneNumberGeocoding"
+  s.version      = "0.9.15"
+  s.summary      = "Short Number Support for libPhoneNumber-iOS"
+  s.description  = <<-DESC
+libPhoneNumberShortNumber for iOS
+iOS library for implementing libPhoneNumber features on short numbers. 
+DESC
+  s.homepage     = "https://github.com/iziz/libPhoneNumber-iOS.git"
+  s.license      = 'Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)'
+  s.authors      = { "rastaarh" => "rastaar@google.com"}
```

Comment:
Maybe add me there too so when you leave Google after the internship, people can still contact someone at the company

Reply:
yup, done.